web/templates/settings/admin_user_list.hbs

```diff
@@ -43,13 +43,17 @@
     {{#if can_modify}}
     <td class="actions">
         <span class="user-status-settings">
-            <button class="button rounded small btn-warning open-user-form" {{#unless is_active}}style="display: none;"{{/unless}} title="{{#if is_bot}}{{t 'Edit bot' }}{{else}}{{t 'Edit user' }}{{/if}}" data-user-id="{{user_id}}">
-                <i class="fa fa-pencil" aria-hidden="true"></i>
-            </button>
+            <div id="edit-button" class="inline-block {{#if (and is_bot cannot_edit)}}tippy-zulip-tooltip{{/if}}" {{#if (and is_bot cannot_edit)}}data-tippy-content="{{t 'This bot cannot be edited.'}}"{{/if}}>
```

Comment:
Sorry I did not notice it earlier, but we should avoid adding same ID for multiple elements. Instead we can add a class like `edit-bot-button-container`. Same changes should be done for deactivate button as well.

Reply:
Isn't the class name suggested is somewhat misleading because this template is also used for rendering users list under organization panel. I think we can remove the id without adding any extra class.

Reply:
Yeah, I think we can avoid any extra class for now since we are not setting the tooltip in the javascript file.